### PR TITLE
feat: search by sample _id and analysis name

### PR DESCRIPTION
**Pull request by maasencioh**
@targos is like this what you said?

### CODE REVIEW

File: configBase.js [hunk 1]
```diff
@@ -29,7 +29,17 @@ module.exports = {
                             emit(doc.$content.samples[i]);
                         }
                     }
-                }
+                },
+                designDoc: 'analysis'
+            },
+            analysisBySampleAndId: {
```
Comment (targos) [inline configBase.js:35]: not sure about this name. why not just `analysisById` ?

Comment (targos) [reply configBase.js:35]: because that's what you emit: the id of the analysis

Comment (maasencioh) [reply configBase.js:35]: but in there there's the uuid of the sample, that's why I though that this name would said more

**Comment by maasencioh** [comment]
@targos currently the `$id` is an array with `[sampleUuid, viewUuid, customId]`, so I'm actually emitting with the sample UUID and the `customId` for the analysis

File: configBase.js (hunk 1 above)
Comment (stropitek) [reply configBase.js:35]: I agree with Michael that `analysisById` seems better. The key you are emitting is the `$id` of the analysis.

**Pull request merged**